tests/components/profiler/test_config_flow.py

```diff
@@ -0,0 +1,45 @@
+"""Test the Profiler config flow."""
+from homeassistant import config_entries, setup
+from homeassistant.components.profiler.const import DOMAIN
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+
+async def test_form_user(hass):
+    """Test we can setup by the user."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] is None
+
+    with patch(
+        "homeassistant.components.profiler.async_setup", return_value=True
+    ) as mock_setup, patch(
+        "homeassistant.components.profiler.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry:
+        result2 = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            {},
+        )
+
+    assert result2["type"] == "create_entry"
+    assert result2["title"] == "Profiler"
+    assert result2["data"] == {}
+    await hass.async_block_till_done()
+    assert len(mock_setup.mock_calls) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6285789)